src/core/lib/security/credentials/tls/openssl_utils.h

```diff
@@ -0,0 +1,27 @@
+//
+// Created by itsemmanuel on 7/16/21.
+// TODO: Replace with appropriate comments.
```

Comment:
Is there a guide I could use to write the appropriate comments at the top including copyrights, licensing, etc.
@ZhenLian @ashithasantosh

Reply:
Just copy the headings of an existing file and update the creation time to be 20221.